gustaf/create/faces.py

```diff
@@ -33,3 +34,129 @@ def box(bounds=[[0, 0], [1, 1]], resolutions=[2, 2]):
     face_mesh = Faces(vertex_mesh.vertices, connectivity)
 
     return face_mesh
+
+def simplexify(quad, backslash=False, alternate=True):
+    """
+    Given quad faces, diagonalize them to turn them into triangles.
+    If quad is counterclockwiese (CCW), triangle will also be CCW and 
+    vice versa.
+    Default diagonalization looks like this:
+    (3) *---* (2)
+        |  /|
+        | / |
+        |/  |
+    (0) *---* (1)
+    resembling 'slash'.
+    If you want to diagonalize the other way, set `backslash=True`.
+    The algorithm will try to alternate the diagonal directions of neighboring
+    elements, if `alternate=True`. This will only work well if the elements are
+    ordered in a suitable manner.
+    Parameters
+    ----------
+    quad: Faces
+    backslash: bool
+    alternate: bool
+    Returns
+    --------
+    tri: Faces
```

Comment:
```suggestion
    ordered in a suitable manner.
    
    Parameters
    ----------
    quad: Faces
    backslash: bool
    alternate: bool
    
    Returns
    --------
    tri: Faces
```
Please add empty lines